shared/macros/10-oval.jinja

```diff
@@ -336,6 +336,21 @@ Generates the :code:`<affected>` tag for OVAL check using correct product platfo
 </ind:textfilecontent54_state>
 {{%- endmacro %}}
 
+{{%- macro oval_line_in_file_state_xccdf_variable(var_name='', id_stem='', datatype='') -%}}
+{{%- set id_stem = id_stem or rule_id -%}}
+<local_variable id="local_var_regex_{{{ id_stem }}}"
```

Comment:
I have discovered that eg. in `build/rhel9/checks/oval/sshd_set_keepalive.xml` this generates 2 local variables that reference the same external variable. It probably isn't a problem.

Reply:
I think I solved it by 444811a0fe3d92d938999c7009a4e1325fb7fc2f